engine/apps/alerts/escalation_snapshot/snapshot_classes/escalation_snapshot.py

```diff
@@ -88,9 +88,7 @@ def executed_escalation_policy_snapshots(self) -> typing.List["EscalationPolicyS
         """
         if self.last_active_escalation_policy_order is None:
             return []
-        elif self.last_active_escalation_policy_order == 0:
-            return [self.escalation_policies_snapshots[0]]
-        return self.escalation_policies_snapshots[: self.last_active_escalation_policy_order]
+        return self.escalation_policies_snapshots[: self.last_active_escalation_policy_order + 1]
```

Comment:
Off-by-one bug making `executed_escalation_policy_snapshots` smaller than it should be (it probably affects the escalation auditor task)